tests/baselines/reference/mappedTypeConcreteTupleHomomorphism.types

```diff
@@ -0,0 +1,50 @@
+=== tests/cases/compiler/mappedTypeConcreteTupleHomomorphism.ts ===
+type A = [1, 2]
+>A : A
+
+type B = {
+>B : ["1", "2"]
+
+    [K in keyof A]: `${A[K]}`
+}
+
+const b: B = ['1', '2']
+>b : ["1", "2"]
+>['1', '2'] : ["1", "2"]
+>'1' : "1"
+>'2' : "2"
+
+type C<T> = {
+>C : C<T>
+
+    [K in keyof T]: [K, T[K]]
+}
+
+type D = {
+>D : [1, 1, 1]

```

Comment:
same here, it's working 🎉 , probably it isn't really different from the previous test case but I've thought that it might be worth checking such case too